src/AppInstallerCLICore/Commands/InstallCommand.cpp

```diff
@@ -128,6 +128,7 @@ namespace AppInstaller::CLI
         else

```

Comment:
**This is outside the scope of the PR**, but could manifest also be a multi-parameter field? It's possible that a user would want to install an entire set of manifests

Should I create a separate issue for that?

Reply:
I hadn't thought about that. I think that'd be a bit harder to implement because manifest is not a positional argument so we'd need to reason about the arg parsing a bit more.

I'd create a separate issue and mention it in the original issue for installing multiple apps in a single command as related.

Reply:
And since installing from manifest is more of a winget dev only feature, it's probably ok to leave this feature out and not make the workflow more complex. It's also harder for debugging if multiple manifests are being installed.